<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uix_information_company</class>
 <widget class="QDialog" name="uix_information_company">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(38,171,147);</string>
   </property>
   <property name="text">
    <string>Informacion de empresa</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Idealist Sans</family>
    </font>
   </property>
   <property name="text">
    <string>R.F.C</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtRFC">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 4px;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtRSocial">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 4px;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Idealist Sans</family>
    </font>
   </property>
   <property name="text">
    <string>Razon social</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>89</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Idealist Sans</family>
    </font>
   </property>
   <property name="text">
    <string>Direcci√≥n</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDireccion">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 4px;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboEstado">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>461</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Idealist Sans</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 4px;
border:1px solid black;</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Idealist Sans</family>
    </font>
   </property>
   <property name="text">
    <string>Estado</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Idealist Sans</family>
    </font>
   </property>
   <property name="text">
    <string>Localidad</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboMunicipio">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>461</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Idealist Sans</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 4px;
border:1px solid black;</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboColonia">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>461</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 4px;
border:1px solid black;</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Idealist Sans</family>
    </font>
   </property>
   <property name="text">
    <string>Municipio</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Idealist Sans</family>
    </font>
   </property>
   <property name="text">
    <string>Codigo Postal</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboColonia_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>461</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 4px;
border:1px solid black;</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Idealist Sans</family>
    </font>
   </property>
   <property name="text">
    <string>Numero ext.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Idealist Sans</family>
    </font>
   </property>
   <property name="text">
    <string>Numero int.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNint">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 4px;
border:1px solid black;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNext">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 4px;
border:1px solid black;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 4px;
border:1px solid black;
background-color:rgb(223,31,41);
color:white;</string>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAceptar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>350</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 4px;
border:1px solid black;
background-color:rgb(237,133,39);
color:white;</string>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtRSocial</tabstop>
  <tabstop>txtRFC</tabstop>
  <tabstop>txtDireccion</tabstop>
  <tabstop>txtNext</tabstop>
  <tabstop>txtNint</tabstop>
  <tabstop>comboEstado</tabstop>
  <tabstop>comboMunicipio</tabstop>
  <tabstop>comboColonia</tabstop>
  <tabstop>comboColonia_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
